### ISSUE TITLE
Wellington needs some love to work with MacOS Sierra

### ISSUE THREAD

**Issue by pierceray** [wellington/wellington#186]
I'm getting errors trying to use wellington while running on Sierra.  We are currently using wellington-bin 1.0.0 to bring in the wellington dependencies.  After a quick chat with @drewwells we are going to have to upgrade wellington and wellington-bin.

```
failed MSpanList_Insert 0x4a069d8 0xbc48d1fecd 0x0
fatal error: MSpanList_Insert
runtime stack:
runtime.MSpanList_Insert(0x473f9d0, 0x4a069d8)
    /usr/local/go142/src/runtime/mheap.c:692 +0x8f
runtime.MHeap_Alloc(0x473f960, 0x1, 0x10000000002, 0x40fac09)
    /usr/local/go142/src/runtime/mheap.c:240 +0x66
runtime.MCentral_CacheSpan(0x47464d8, 0x4a05d40)
    /usr/local/go142/src/runtime/mcentral.c:85 +0x167
runtime.MCache_Refill(0x4a00000, 0x2, 0x4a05d40)
    /usr/local/go142/src/runtime/mcache.c:90 +0xa0
goroutine 1 [running, locked to thread]:
runtime.switchtoM()
    /usr/local/go142/src/runtime/asm_amd64.s:198 fp=0xc2080b4678 sp=0xc2080b4670
runtime.mallocgc(0x4, 0x44e8ce0, 0x1, 0x0)
    /usr/local/go142/src/runtime/malloc.go:147 +0x1bb fp=0xc2080b4728 sp=0xc2080b4678
runtime.newobject(0x44e8ce0, 0x4a11d68)
    /usr/local/go142/src/runtime/malloc.go:353 +0x49 fp=0xc2080b4750 sp=0xc2080b4728
runtime.convT2I(0x44e8ce0, 0x44926a0, 0x4738028, 0xc2080b47cc, 0x0, 0x0)
    /usr/local/go142/src/runtime/iface.go:163 +0x130 fp=0xc2080b4798 sp=0xc2080b4750
image.(*RGBA).At(0xc2087baf00, 0xd2, 0x4f3, 0x0, 0x0)
    /usr/local/go142/src/image/image.go:75 +0xa4 fp=0xc2080b47d8 sp=0xc2080b4798
image/png.writeImage(0x4a13df8, 0xc2087baf80, 0x4a138b0, 0xc2087baf00, 0xb, 0xffffffffffffffff, 0x0, 0x0)
    /usr/local/go142/src/image/png/writer.go:379 +0xf47 fp=0xc2080b4b00 sp=0xc2080b47d8
image/png.(*encoder).writeIDATs(0xc2082bc480)
    /usr/local/go142/src/image/png/writer.go:446 +0x151 fp=0xc2080b4bb8 sp=0xc2080b4b00
image/png.(*Encoder).Encode(0xc20807df60, 0x4a11fd8, 0xc2080aeb40, 0x4a138b0, 0xc2087baf00, 0x0, 0x0)
    /usr/local/go142/src/image/png/writer.go:527 +0x387 fp=0xc2080b4d00 sp=0xc2080b4bb8
image/png.Encode(0x4a11fd8, 0xc2080aeb40, 0x4a138b0, 0xc2087baf00, 0x0, 0x0)
    /usr/local/go142/src/image/png/writer.go:476 +0x72 fp=0xc2080b4d40 sp=0xc2080b4d00
github.com/wellington/spritewell.(*ImageList).Combine(0xc2080aeb40, 0x0, 0x0, 0x0, 0x0)
    /Users/drew/src/github.com/wellington/spritewell/sprite.go:288 +0x498 fp=0xc2080b4ec8 sp=0xc2080b4d40
github.com/wellington/wellington/handlers.SpriteMap(0x45067e0, 0xc20884ce00, 0x86b9f80, 0xc20802c1f0, 0x0, 0x0)
    /Users/drew/src/github.com/wellington/wellington/handlers/spriting.go:221 +0x867 fp=0xc2080b5190 sp=0xc2080b4ec8
github.com/wellington/go-libsass.func·002(0x45067e0, 0xc20884ce00, 0x86b9f80, 0xc20802c1e8, 0x0, 0x0)
    /Users/drew/src/github.com/wellington/go-libsass/func.go:23 +0xf0 fp=0xc2080b5200 sp=0xc2080b5190
github.com/wellington/go-libsass/libs.GoBridge(0x86b9f80, 0xc208060570, 0x2)
    /Users/drew/src/github.com/wellington/go-libsass/libs/export.go:29 +0x7b fp=0xc2080b5278 sp=0xc2080b5200
runtime.call32(0x7fff5fbfd990, 0x7fff5fbfda18, 0x18)
    /usr/local/go142/src/runtime/asm_amd64.s:402 +0x45 fp=0xc2080b52a0 sp=0xc2080b5278
runtime.cgocallbackg1()
    /usr/local/go142/src/runtime/cgocall.go:239 +0x12a fp=0xc2080b5300 sp=0xc2080b52a0
runtime.cgocallbackg()
    /usr/local/go142/src/runtime/cgocall.go:193 +0x6e fp=0xc2080b5330 sp=0xc2080b5300
runtime.cgocallback_gofunc(0x412b9ff, 0x40f099d, 0x4002530)
    /usr/local/go142/src/runtime/asm_amd64.s:766 +0x57 fp=0xc2080b5340 sp=0xc2080b5330
asmcgocall()
    /usr/local/go142/src/runtime/asm_amd64.s:649 +0x37 fp=0xc2080b5348 sp=0xc2080b5340
runtime.asmcgocall_errno(0x4002530, 0xc2080b5398)
    /usr/local/go142/src/runtime/asm_amd64.s:626 +0xf fp=0xc2080b5350 sp=0xc2080b5348
runtime.cgocall_errno(0x4002530, 0xc2080b5398, 0x0)
    /usr/local/go142/src/runtime/cgocall.go:131 +0x10d fp=0xc2080b5378 sp=0xc2080b5350
github.com/wellington/go-libsass/libs._Cfunc_sass_compiler_parse(0x9d00810, 0x0)
    github.com/wellington/go-libsass/libs/_obj/_cgo_gotypes.go:147 +0x43 fp=0xc2080b5398 sp=0xc2080b5378
github.com/wellington/go-libsass/libs.SassCompilerParse(0x9d00810)
    /Users/drew/src/github.com/wellington/go-libsass/libs/wrap.go:198 +0x28 fp=0xc2080b53b0 sp=0xc2080b5398
github.com/wellington/go-libsass.(*Context).FileCompile(0xc20884ce00, 0x7fff5fbff3c7, 0x18, 0x4a11bf0, 0xc20802c168, 0x0, 0x0)
    /Users/drew/src/github.com/wellington/go-libsass/context.go:113 +0x175 fp=0xc2080b54d0 sp=0xc2080b53b0
github.com/wellington/wellington.LoadAndBuild(0x7fff5fbff3c7, 0x18, 0xc208010230, 0xc20801e940, 0x0, 0x0)
    /Users/drew/src/github.com/wellington/wellington/build.go:77 +0xc26 fp=0xc2080b58e0 sp=0xc2080b54d0
main.Run(0x471e800, 0xc20800b880, 0x28, 0x34)
    /Users/drew/src/github.com/wellington/wellington/wt/main.go:348 +0x2f94 fp=0xc2080b5da8 sp=0xc2080b58e0
github.com/spf13/cobra.(*Command).execute(0x471e800, 0xc20800b180, 0x34, 0x38, 0x0, 0x0)
    /Users/drew/src/github.com/spf13/cobra/command.go:477 +0x3d4 fp=0xc2080b5e70 sp=0xc2080b5da8
github.com/spf13/cobra.(*Command).Execute(0x471ece0, 0x0, 0x0)
    /Users/drew/src/github.com/spf13/cobra/command.go:551 +0x414 fp=0xc2080b5f78 sp=0xc2080b5e70
main.main()
    /Users/drew/src/github.com/wellington/wellington/wt/main.go:155 +0x34 fp=0xc2080b5f98 sp=0xc2080b5f78
runtime.main()
    /usr/local/go142/src/runtime/proc.go:63 +0xf3 fp=0xc2080b5fe0 sp=0xc2080b5f98
runtime.goexit()
    /usr/local/go142/src/runtime/asm_amd64.s:2232 +0x1 fp=0xc2080b5fe8 sp=0xc2080b5fe0
goroutine 17 [syscall, locked to thread]:
runtime.goexit()
    /usr/local/go142/src/runtime/asm_amd64.s:2232 +0x1
```

**Comment by drewwells** [comment]
It looks like this is related to OS X Sierrra breaking the go toolchain. I'm upgrading my laptop to Sierra and testing this with Go 1.7.1

**Comment by pierceray** [comment]
Version 1.0.3 released

Fixed by 58493fa911555135fd804b97320f915a24a589e0

**Comment by IrishAdo** [comment]
Just trying to get the project out at the minute so will test in a couple of days.  I've realised that I didn't give any build information  here is the content of the Gruntfile.js we are using ;-) just happy I figured out how to get it to add the comments to the config file.

module.exports = function(grunt) {
  // Config.
  grunt.initConfig({
    pkg: grunt.file.readJSON('package.json'),
    watch: {
      scripts: {
        files: ['./js/src/*.js', '!./js/script.gen.js'],
         tasks: ['jshint', 'concat:tp_boilerplate']
        //  tasks: [ 'concat:tp_boilerplate']
      },
      sass: {
          files: [ 'sass/**/*.scss' ],
          tasks: [ 'wellington:dist' ]
      }
    },
    concat: {
      tp_boilerplate: {
        src: './js/src/**/*.js',
        dest: './js/script.gen.js'
      }
    },
    jshint: {
      options: {
        jshintrc: '.jshintrc',
        reporterOutput: ""
      },
      all: ['js/src/*.js', '!js/script.gen.js']
    },
    wellington: {
        dist: {
            src: [
                'sass/**/*.scss',   // Build all .scss files
                '!sass/**/_*.scss'  // Don't build files starting with "_" into css files
            ],
            options: {
              comment:true,
              style: 'expanded',      // The style of output to be used.
              p: 'sass',       // The base folder that contains the sass.
              b: '.',  // The output folder for the built css.
              d: 'images',    // The input image folder
              gen: 'build/im'  // The output folder for the generated sprite files.
            }
        },
        options: {
            style: 'nested'      // The style of output to be used.
        }
    },

  });

  // Load the tasks...
  grunt.loadNpmTasks('grunt-contrib-watch');
  grunt.loadNpmTasks('grunt-contrib-jshint');
  grunt.loadNpmTasks('grunt-contrib-concat');
  grunt.loadNpmTasks('grunt-wellington');
  // Default task.
  grunt.registerTask('default', 'watch');

}
